### ISSUE TITLE
Import Calendars from iCal file (.ics file)

### ISSUE THREAD

**Issue by mouhb** [xwikisas/application-mocca-calendar#114]
According to the "Calendar Application (Pro)" application [documentation](https://store.xwiki.com/xwiki/bin/view/Extension/CalendarApplication#documentation) it is only possible to create calendars/events one by one manually on the wiki, so, the objective of this ticket is to improve the application to be able to import calendars using an [iCal](https://en.wikipedia.org/wiki/ICalendar) file (.ics file).
Basically, the user should be able to create calendars/events on the wiki by uploading an ics (file) or by providing an URL to .ics file.
